src/main/server.ts

```diff
@@ -33,26 +36,20 @@ function getSSLOptions (): any {
   }
 }
 
-function forkListenerProcesses (numberOfCores: number) {
-  for (let i = 0; i < numberOfCores; i++) {
-    cluster.fork()
+function applicationPort (): any {
+  let port: any = process.env.PORT
```

Comment:
If we cannot restrict this to number alone lets do `number | string`.

Reply:
looks like this works now, I'm pretty sure before it had to be a number or any.